app/controllers/api/v1/config_templates_controller.rb

```diff
@@ -95,7 +95,7 @@ def resource_class
       end
 
       def deprecated
-        ::ActiveSupport::Deprecation.warn('Config templates were renamed to provisioning templates')
+        Foreman::Deprecation.deprecation_warning("1.11", "Config templates were renamed to provisioning templates")
```

Comment:
I'm unsure if we should set a deadline for a single controller in the v1 API, since part of the point of it is that it's versioned and won't change.  If we remove it, it seems like we'd break this entirely as there's no v1 replacement?